### PR TITLE
chore(doc-site): add doc site page for migration utility

### PR DESCRIPTION
**Pull request by ssjagad**
## Overview
Add documentation for migrateDashboard utility function

## Legal
This project is available under the [Apache 2.0 License](http://www.apache.org/licenses/LICENSE-2.0.html).

### CODE REVIEW

File: packages/doc-site/stories/utilities/migrateDashboard.mdx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    import { Dashboard } from '@iot-app-kit/dashboard';
+
+    const client = new IoTSiteWise({
+        credentials: awsCredentials,
+        region: awsRegion,
+    });
+
+    const MyDashboard = () => {
+        const options = {
+            params: {dashboardId: "sitewiseMonitorDashboardId"}
```
Comment (tracy-french) [inline packages/doc-site/stories/utilities/migrateDashboard.mdx:29]: Typo: API is parameters, not params, right?

File: packages/doc-site/stories/utilities/migrateDashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            requestFns: client
```
Comment (tracy-french) [inline packages/doc-site/stories/utilities/migrateDashboard.mdx:30]: iotSiteWiseClient

File: packages/doc-site/stories/utilities/migrateDashboard.mdx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        return <Dashboard dashboardConfig={appKitDashboardConfiguration} />;
+    };
+```
+
+## describeDashboard parameters
+
+**options (object, required)** 
+These properties are used to specify which dashboard is to be migrated from Sitewise Monitor and the authenticated API call to allow it.
+
+- **params (object, required)**
```
Comment (tracy-french) [inline packages/doc-site/stories/utilities/migrateDashboard.mdx:43]: parameters

File: packages/doc-site/stories/utilities/migrateDashboard.mdx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    
+    - **dashboardId (string, required)**: Specifies the id of the Sitwise Monitor dashboard to be converted to into an Application Kit dashboard.
+
+- **client (required)**: An instance of the AWS Sitewise Client. The following methods are required for this component.
```
Comment (tracy-french) [inline packages/doc-site/stories/utilities/migrateDashboard.mdx:47]: iotSiteWiseClient

File: packages/dashboard/src/migration/convert-monitor-to-app-defintion.spec.ts [hunk 5]
```diff
@@ -27,10 +27,10 @@ describe('Dashboard definition conversion', () => {
...
     });
 
-    const applicationDefinition = await migrateDashboard(
-      { dashboardId: 'test-id' },
-      { describeDashboard: describeDashboard }
-    );
+    const applicationDefinition = await migrateDashboard({
+      parameters: { dashboardId: 'test-id' },
+      iotSiteWiseClient: { describeDashboard: describeDashboard },
+    });
```
Comment (tracy-french) [inline packages/dashboard/src/migration/convert-monitor-to-app-defintion.spec.ts:33]: Nice! :)

**Review by tracy-french — DISMISSED** [review]

File: packages/doc-site/stories/utilities/migrateDashboard.mdx (hunk 1 above)
Comment (ssjagad) [reply packages/doc-site/stories/utilities/migrateDashboard.mdx]: yep, forgot to update the docsite with change. done now!

**Review by chejimmy — APPROVED** [review]
LGTM!

**Review by corteggiano — APPROVED** [review]

**Pull request merged**